ci/run-tests.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -eo pipefail
+
+ROOT="$(cd "$(dirname "$0")/.." && pwd)"
+
+main() {
+	echo "** Starting xvfb"
+	export DISPLAY=:99
+	/usr/bin/Xvfb $DISPLAY &
+	echo "** Started xvfb"
```

Comment:
As some point I should lock you and Paddy in a room and force you to fight it out, winner gets to decide the tabs vs spaces policy :P